packages/tools/client-debugger/client-debugger/src/test/Utilities.ts

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { EventEmitter } from "events";
+
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import {
+    IAudience,
+    IAudienceOwner,
+    IContainer,
+    IContainerEvents,
+    IDeltaManager,
+    IDeltaManagerEvents,
+} from "@fluidframework/container-definitions";
+import { ConnectionState } from "@fluidframework/container-loader";
+import {
+    IClient,
+    IDocumentMessage,
+    ISequencedDocumentMessage,
+} from "@fluidframework/protocol-definitions";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/**
+ * Mock {@link @fluidframework/container-definitions#IDeltaManager} for use in tests.
+ */
+class MockDeltaManager
+    extends TypedEventEmitter<IDeltaManagerEvents>
+    implements
+        Partial<
+            Omit<IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>, "on" | "off" | "once">
+        >
+{
+    public hasCheckpointSequenceNumber = true;
```

Comment:
When will we use CheckpointSequenceNumber?

Reply:
Whoops, I forgot to push a change. We don't need this type at all.